### PR TITLE
PEP and bug fix

### PR DESCRIPTION
**Pull request by anthonybu**

### CODE REVIEW

File: python/generational/decisions.py [hunk 1]
```diff
@@ -36,7 +36,8 @@
     "mating": 2     # need energy to mate, as well as attraction
 }
 
-class Work():
+
+class Work:
```
Comment (anthonybu) [inline python/generational/decisions.py:40]: unlike function, class doesn't need the () after name

File: python/generational/decisions.py [hunk 2]
```diff
@@ -72,7 +75,7 @@ def __init__(self, id):
         self.mother = None
         self.partner = None
         self.children = 0
-        self.sex = id % 2   # 0 for female, 1 for male
+        self.sex = _id % 2   # 0 for female, 1 for male
```
Comment (anthonybu) [inline python/generational/decisions.py:78]: id is a build in keyword, I'm changing this into a internal variable

File: python/generational/decisions.py [hunk 3]
```diff
@@ -117,9 +120,9 @@ def __init__(self, id):
         self.threshold_reward = 0                              # used to scale fitness
         ## Action Tables
         self.target_bias = {
-            "working": {[self.satisfaction_score, self.energy_score, self.money_score], 1},
-            "eating": {[self.satisfaction_score, self.energy_score, self.money_score], 1},
-            "mating": {[self.satisfaction_score, self.energy_score, self.money_score], 1}
+            "working": ((self.satisfaction_score, self.energy_score, self.money_score), 1),
```
Comment (anthonybu) [inline python/generational/decisions.py:123]: {} is for dictionary or set, I'm assuming you want to just store those values, so I changed to tuples, but if those value changes, then you may want to use list instead -- tuples are not mutable

File: python/generational/decisions.py [hunk 4]
```diff
@@ -191,5 +196,6 @@ def main():
         pygame.display.update()
         clock.tick(15)
 
+
 if __name__ == "__main__":
-    main()
\ No newline at end of file
+    main()
```
Comment (anthonybu) [inline python/generational/decisions.py:201]: acroding to PEP, py file should be ending with a blank line

File: python/generational/decisions.py [hunk 5]
```diff
@@ -36,7 +36,8 @@
     "mating": 2     # need energy to mate, as well as attraction
 }
 
-class Work():
+
```
Comment (anthonybu) [inline python/generational/decisions.py:39]: acroding to PEP, two line breaks between classes

File: python/generational/decisions.py (hunk 2 above)
Comment (alephpt) [reply python/generational/decisions.py:78]: I'm going to change this to 'identity' to be more thorough

File: python/generational/decisions.py (hunk 3 above)
Comment (alephpt) [reply python/generational/decisions.py:123]: I'm not sure if I want to use an enum, dictionary or a list yet, but it needs to be mutable. I will test a few ideas out.

File: python/generational/decisions.py (hunk 4 above)
Comment (alephpt) [reply python/generational/decisions.py:201]: good to know! 👍

File: python/generational/decisions.py (hunk 5 above)
Comment (alephpt) [reply python/generational/decisions.py:39]: thanks for the information 👍  finally, a python standard I like 😄 🎉

**Pull request merged**